sonar-python-plugin/src/main/java/org/sonar/plugins/python/IpynbNotebookParser.java

```diff
@@ -68,7 +70,7 @@ public ParseResult parseNotebook() throws IOException {
       }
     }
 
-    return new ParseResult(inputFile, aggregatedSource.toString(), locationMap);
+    return new ParseResult(inputFile, aggregatedSource.toString(), locationMap, colOffSet);
```

Comment:
I guess we could remove the colOffSet here, as all the data is in the IPythonLocation